### ISSUE TITLE
Support for Spring Cloud Gateway

### ISSUE THREAD

**Issue by wtatum** [codecentric/spring-boot-admin#993]
We've been using [Spring Cloud Gateway](https://cloud.spring.io/spring-cloud-static/spring-cloud-gateway/2.1.0.M1/single/) as our API gateway, as well as for some internal routing between services. It seems like a great up and comer to Zuul, and ties in with Spring Boot and Spring Cloud really nicely.

Are there any plans to support the /actuator/gateway endpoints which are exposed by Spring Cloud Gateway? Unfortunately the reference manual doesn't yet include many details about the actuator API, but it's fairly straightforward and works well over direct GET/POST.

**Comment by joshiste** [comment]
I don't have this planned and as I am not a SC-Gateway user it will probably never be on my todolist.
But contributions are alway welcome.

**Comment by wtatum** [comment]
Okay, thanks for the quick reply.

**Comment by zqr95518** [comment]
There's a simple way to solve this. Edit the index.html and change all uri of js, css, etc to reletive path.

**Comment by joshiste** [comment]
@zqr95518 Your post is totally unrelated to this issue

**Comment by zqr95518** [comment]
@joshiste No, it may releted. I access the page from gateway with http://{gateway:port}/admin/login. And the js and css not get correctly due to the url become http://{gateway:port}/xxxxxx.js but it should be http://{gateway:port}/admin/xxxxxx.js.

**Comment by zqr95518** [comment]
Oh, it's my fault. Although we both need them. ☺

**Comment by mzorba** [comment]
Work is in progress on this. You can expect pull request very soon.

**Comment by joshiste** [comment]
looks nice

**Comment by valera7979** [comment]
It doesn't work for me. I have SpringBootAdmin application. When I trying to path throw gateway server on Spring admin server I see that Spring-admin server tries get front files  http://{gateway:port}/xxxxxx.js instead http://{gateway:port}/admin/xxxxxx.js exactly as described by @zqr95518 earlier.

**Comment by joshiste** [comment]
@valera7979 the issues is **not** about running SBA behind spring cloud gateway. But this chapter might be worthy to read: https://codecentric.github.io/spring-boot-admin/current/#_running_behind_a_front_end_proxy_server
